plugins/available/cmd-duration.plugin.bash

```diff
@@ -0,0 +1,46 @@
+cite about-plugin
+about-plugin 'keep track of the moment when the last command started, to be able to compute its duration'
+
+# Define tmp dir and file
+COMMAND_DURATION_TMPDIR="${TMPDIR:-/tmp}"
+COMMAND_DURATION_FILE="$COMMAND_DURATION_TMPDIR/bashit_theme_execution_$BASHPID"
+
+COMMAND_DURATION_ICON='  '
+
+trap _command_duration_delete_temp_file EXIT HUP INT TERM
+
+_command_duration_delete_temp_file() {
```

Comment:
Any reason why you're using two different styles of `function` declarations (here and line #22 for example)?

Reply:
no, fixed by 22aae07d0d2efc96b2ac5a64c04bdf867e4c35e6